pandas/_libs/tslibs/period.pyx

```diff
@@ -1295,6 +1295,25 @@ cdef class _Period(object):
 
     @property
     def second(self):
+        """
+        Get second of the minute component of the Period.
+
+        Returns
+        -------
+        int 
+            The second as an integer, between 0 and 59.
```

Comment:
Already say second is an int above so no need to restate here. Maybe better worded as "The second of the Period (ranges from 0 to 59)"

Reply:
yup sure :+1